tests/lib/openstack_query/mappings/test_user_mapping.py

```diff
@@ -0,0 +1,47 @@
+from enums.query.props.user_properties import UserProperties
+from enums.query.query_presets import QueryPresetsGeneric
+from openstack_query.handlers.server_side_handler import ServerSideHandler
+from openstack_query.mappings.user_mapping import UserMapping
+from openstack_query.runners.user_runner import UserRunner
+
+
+def test_get_runner_mapping():
+    """
+    Tests get runner mapping returns a user runner
+    """
+    assert UserMapping.get_runner_mapping() == UserRunner
+
+
+def test_get_prop_mapping():
+    """
+    Tests get prop mapping returns a server properties
+    """
+    assert UserMapping.get_prop_mapping() == UserProperties
+
+
+def test_get_server_side_handler_returns_correct_type():
+    """
+    Tests get server side handler returns a server side handler
+    """
+    assert isinstance(UserMapping.get_server_side_handler(), ServerSideHandler)
+
+
+def test_server_side_handler_mappings_equal_to():
+    """
+    Tests server side handler mappings are correct, and line up to the expected
+    server side params for equal to params
+    """
+    handler = UserMapping.get_server_side_handler()
+    mappings = {
+        UserProperties.USER_DOMAIN_ID: "domain_id",
+        UserProperties.USER_NAME: "name",
+        UserProperties.USER_ID: "id",
+    }
+
+    supported_props = handler.get_supported_props(QueryPresetsGeneric.EQUAL_TO)
+    assert all(key_to_check in supported_props for key_to_check in mappings)
+    for prop, expected in mappings.items():
+        user_filter = handler.get_filters(
+            QueryPresetsGeneric.EQUAL_TO, prop, {"value": "test"}
+        )
+        assert user_filter == {expected: "test"}
```

Comment:
Since this mechanism is the same, lets create a helper file with this as a method where we take an expected dict and the instance under test

Since we'll have to duplicate this for every mapping file